### PR TITLE
Add kmd_rewards_info API call

### PR DESCRIPTION
**Pull request by sergeyboyko0791**

### CODE REVIEW

**Comment by sergeyboyko0791** [comment]
@gcharang please review the PR

**Comment by gcharang** [comment]
hi, is this rpc available in the mm2 branch of AtomicDEX-API?

**Comment by sergeyboyko0791** [comment]
Corresponding PR is approved but is not merged yet

File: docs/basic-docs/atomicdex/atomicdex-api.md [hunk 1]
```diff
@@ -1279,6 +1279,91 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
 
 </div>
 
+## kmd\_rewards\_info
+
+**kmd_rewards_info**
+
+The `kmd_rewards_info` method returns rewards info of unspent outputs.
```
Comment (artemii235) [inline docs/basic-docs/atomicdex/atomicdex-api.md:1286]: Please add the note that this method only works when KMD coin is activated.

Comment (sergeyboyko0791) [reply docs/basic-docs/atomicdex/atomicdex-api.md:1286]: Done

**Comment by artemii235** [comment]
@gcharang The corresponding API PR is merged, the docs can be published on website, thanks!

**Pull request merged**